posthog/hogql_queries/query_runner.py

```diff
@@ -39,6 +40,8 @@ class QueryResponse(BaseModel, Generic[DataT]):
     )
     result: DataT
     timings: Optional[List[QueryTiming]] = None
+    types: Optional[Any] = None
+    columns: Optional[Any] = None
```

Comment:
Let's try to avoid `Any`-s

```suggestion
    types: Optional[List[str]] = None
    columns: Optional[List[str]] = None
```